src/hooks/useVenuePoll.ts

```diff
@@ -0,0 +1,60 @@
+import { useMemo, useCallback } from "react";
+import firebase from "firebase/app";
+
+import { voteInVenuePoll, createVenuePoll, deleteVenuePoll } from "api/poll";
+
+import { PollMessage, PollValues, Vote } from "types/chat";
+
+import { useVenueId } from "./useVenueId";
+import { useUser } from "./useUser";
+
+export const useVenuePoll = () => {
+  const venueId = useVenueId();
+  const { user } = useUser();
+
+  const userId = user?.uid;
+
+  const voteInPoll = useCallback(
+    (votes: Vote[], pollId: string) => {
+      if (!venueId) return;
+
+      voteInVenuePoll({ venueId, votes, pollId });
+    },
+    [venueId]
+  );
+
+  const createPoll = useCallback(
+    (pollValues: PollValues) => {
+      if (!venueId || !userId) return;
+
+      const poll: PollMessage = {
+        poll: pollValues,
+        from: userId,
+        text: "poll",
+        votes: [],
+        ts_utc: firebase.firestore.Timestamp.now(),
+      };
```

Comment:
use buildMessage here

Reply:
fixed in 5de120d738c7bd7cbad0071ae5aa5716dbe4a8ec